tools/clang/test/HLSLFileCheck/hlsl/objects/RayQuery/wave_intrinsicAttributeCollision.hlsl

```diff
@@ -0,0 +1,46 @@
+// RUN: %dxc -T ps_6_5 -E PS -fcgl %s | FileCheck %s
+// RUN: %dxc -T ps_6_5 -E PS %s | FileCheck %s -check-prefix=CHECKDXIL
```

Comment:
It seems we allow QuadAny with shader models earlier than 6.7 😣

We shouldn't though. One of us should file a bug for that and this test should be updated to require 6.7

Reply:
Don't we have a fallback for QuadAny on shader models earlier than 6.7?

Reply:
Made an issue here:
https://github.com/microsoft/DirectXShaderCompiler/issues/5521